src/jarabe/journal/journalactivity.py

```diff
@@ -241,12 +255,45 @@ def _session_manager_shutdown_cb(self, event):
     def can_close(self):
         return False
 
-    def _setup_main_view(self):
-        self._main_toolbox = MainToolbox()
-        self._edit_toolbox = EditToolbox(self)
-        self._main_view = Gtk.VBox()
-        self._main_view.set_can_focus(True)
-
+    def _create_add_new_entry(self):
+        self._add_new_box = Gtk.Box(orientation = Gtk.Orientation.HORIZONTAL)
```

Comment:
Can you make the add new box into a class?  I see this function copied around a lot.